projects/angular-auth-oidc-client/src/lib/validation/jwk-window-crypto.service.spec.ts

```diff
@@ -44,9 +44,9 @@ describe('JwkWindowCryptoService', () => {
     });
   });
 
-  beforeEach(() => {
+  beforeEach(async () => {
```

Comment:
use waitforasync

Reply:
@FabianGosebrink Thanks for the feedback! How about now? :smile: